Makefile

```diff
@@ -163,7 +163,7 @@ _build_containerized:
 _build_containerized_aio:
 	@if [ -z '$(CTR_CMD)' ] ; then echo '!! ERROR: containerized builds require podman||docker CLI, none found $$PATH' >&2 && exit 1; fi
 	echo BIN_TIMESTAMP==$(BIN_TIMESTAMP)
-	$(CTR_CMD) build -t $(IMAGE_REPO_AIO):$(SOURCE_GIT_TAG)-linux-$(IPTABLES)-$(ARCH) \
+	$(CTR_CMD) build -t $(IMAGE_REPO_AIO):$(SOURCE_GIT_TAG)-linux-$(ARCH) \
```

Comment:
Hmm, without this we cannot support older systems like the ubuntu on jetson where they run on iptables kernel.

This is also important for older rhel7 testers.

I'd keep/build the different ones for now.

Reply:
The iptables variant is only specified for arm64, for an internal edge case.  I'm okay with leaving it for now.  IMO though using `nft` to reflect the common deployment path is confusing.